cypress/support/helpers/runTestsForPage.js

```diff
@@ -97,40 +97,44 @@ const runTestsForPage = ({
           }
         });
 
-        // Switch to AMP page URL (NB all our pages have AMP variants)
+        // Switch to AMP page URL (NB only some of our pages have AMP variants)
         describe(`${pageType} - ${currentPath} - AMP`, () => {
-          before(() => {
-            Cypress.env('currentPath', currentPath);
-
-            visitPage(getAmpUrl(currentPath), pageType);
-          });
-
-          const testArgs = {
-            service,
-            pageType,
-            variant: config[service].variant,
-            isAmp: true,
-          };
-
-          // Enables overriding of the smoke test values in the config/settings.js file
-          testsThatAlwaysRunForAllPages(testArgs);
-          // Page specific tests
-          testsThatAlwaysRunForAMPOnly(testArgs);
-          testsThatAlwaysRun(testArgs);
+          // TC2 MAPs are not loading the media player which causes the tests to fail when the page is visited, this is accepted behaviour
+          // The substring '/20' is common to all the TC2 MAPs test URLs in the settings and we will not be adding more
+          if (!currentPath.includes('/20')) {
```

Comment:
Would a regex be better here? Or perhaps as we discussed earlier, it's a known pattern, and is never going to change, so this is good enough to get the tests passing.

Reply:
It would definitely be better to have regex if we were ever going to add any more TC2 URLs in the cypress config, as not all in existence would have '/20' in them I expect, like if they are older than the year 2000. But as we won't be adding any more (unlikely?) I think this is sufficient at the moment.

Reply:
Yes, that's fair, and in the spirit of getting this fixed quickly, let's not worry! 🚢 🚢 🚢 🚢